application-tour-test/application-tour-test-tests/src/test/it/org/xwiki/contrib/tour/test/ui/TourApplicationTest.java

```diff
@@ -88,7 +89,7 @@ private void setUpTour() throws Exception
         objectEditPage.clickSaveAndView();
         
         tourHomePage = TourHomePage.gotoPage();
-        assertTrue(tourHomePage.getTours().contains(new TourFromLivetable("Test", "Main.WebHome", true)));
+        assertTrue(tourHomePage.getTours().contains(new TourFromLivetable("Test", "Tour.WebHome", true, "TourCode.TestClass")));
```

Comment:
You should not replace the old test but create a new one. We still want to test the old feature (with targetPage).